### PR TITLE
Add my pin to the map

### PR DESCRIPTION
**Pull request by oswvld**
Prova

### CODE REVIEW

**Comment by kamitsharma05** [comment]
please unsubscribe me from this notifications.

File: oswvld.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: oswvld
+latitude: 35.919789
+longitude: -88.758949
+---
```
Comment (hectorsector) [inline oswvld.json]: @oswvld just make sure this file is inside the `_pins` folder. We made [a video](https://vimeo.com/201011288) that will help you resolve this.

**Comment by hectorsector** [comment]
> please unsubscribe me from this notifications.

@kamitsharma05 you're probably set to watching this repo, you can unsubscribe by switching yourself to not watching, or unsubscribing from this particular conversation using the "Unsubscribe" button. Here's a screenshot that might help.

screen shot 2017-07-28 at 9 37 07 am

**Comment by crichID** [comment]
@oswvld I went ahead and fixed the filename in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/9013/commits/ab030da629cf100a5cdc5e8b05dc06eacf5cdd14. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by August 4th, I will go ahead and do it just to keep the repository clean!

**Pull request merged**